tests/unit_tests/bittensor_tests/test_neuron.py

```diff
@@ -237,89 +237,6 @@ def exit_early(*args, **kwargs):
         # Should try to register the neuron
         mock_register.assert_called_once()
 
-def test_templateminer_reregister_flag_false_exit():
```

Comment:
i think this is sth vune added recently, and it checkes for registration, so maybe convert that for sth core_server?

Reply:
same for the tst below